mopidy_spotify/search.py

```diff
@@ -36,19 +42,42 @@ def search(config, session, query=None, uris=None, exact=False):
         logger.info('Spotify search aborted: Spotify is offline')
         return models.SearchResult(uri=uri)
 
-    sp_search = session.search(
-        sp_query,
-        album_count=config['search_album_count'],
-        artist_count=config['search_artist_count'],
-        track_count=config['search_track_count'])
-    sp_search.load()
+    search_count = max(
+        config['search_album_count'],
+        config['search_artist_count'],
+        config['search_track_count'])
+
+    if search_count > 50:
+        logger.warn(
+            'Spotify currently allows maximum 50 search results of each type. '
+            'Please set the config options `search_album_count`, '
```

Comment:
s/config options/config values/ to keep the terminology consistent. Feel free to drop the backticks and instead prefix with `spotify/`.